markdown-pages/zh/tidb/_docHome.md

```diff
@@ -20,6 +20,8 @@ hide_leftNav: true
 
 | 版本        | 归档文档链接                                             | 归档日期 |
 | ---------- | ------------------------------------------------------ | ----------------- |
+| v7.2 (DMR) | [TiDB v7.2 文档](https://docs-archive.pingcap.com/zh/tidb/v7.2) | 2023 年 10 月 20 日 |
+| v3.0       | [TiDB v3.0 文档](https://docs-archive.pingcap.com/zh/tidb/v3.0) | 2023 年 10 月 20 日 |
```

Comment:
The same

Reply:
done in https://github.com/pingcap/docs-staging/pull/54/commits/dbd0bc1926a45642862f3389958e045ec4ae188c.